uni/ulsp/database.icn

```diff
@@ -143,70 +167,72 @@ class LSPDB(
                file_without_ext := tab(upto("."))
             }
 
-            package_db[pack.getName()]["files"][file_without_ext] := table()
-            if /package_db[pack.getName()]["files"][file_without_ext]["imports"] then package_db[pack.getName()]["files"][file_without_ext]["imports"] := []
+
+            file_table[file_without_ext] := table()
+
+            /file_table[file_without_ext]["imports"]  := (import_table := [])
             every import_in_file := file_in_pack.getImports().get() do {
-               put(package_db[pack.getName()]["files"][file_without_ext]["imports"], import_in_file.getName())
+               put(import_table, import_in_file.getName())
             }
```

Comment:
I see one problem with using `import_table` compared to the old code. That variable is only going to be initialized if the this is the first time `file_table[file_without_ext]["imports"] ` is used. 
this really should be 
```
/file_table[file_without_ext]["imports"]  :=  []
import_table := file_table[file_without_ext]["imports"]
```